app/subscriber/src/store/hooks/subscriber/useUsers.ts

```diff
@@ -12,23 +12,17 @@ interface IUserController {
 export const useUsers = (): IUserController => {
   const api = useApiUsers();
   const dispatch = useAjaxWrapper();
-  const [state, store] = useAdminStore();
+  const [, store] = useProfileStore();
 
   const controller = React.useMemo(
     () => ({
       updateUser: async (model: IUserModel) => {
```

Comment:
I should also change this to `useProfile` to better reflect what it is doing.  But I've already made so many changes in this PR.